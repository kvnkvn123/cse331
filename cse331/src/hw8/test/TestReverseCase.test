# Verify that the program finds and correctly reverse output
# of the previous test: the shortest path from
# CSE to PAB.
r
PAB
CSE

# All of your test scripts should end with q to terminate the program.
q