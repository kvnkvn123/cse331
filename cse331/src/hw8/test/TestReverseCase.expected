Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Verify that the program finds and correctly reverse output
# of the previous test: the shortest path from
# CSE to PAB.
Abbreviated name of starting building: Abbreviated name of ending building: Path from Physics/Astronomy Building to Paul G. Allen Center for Computer Science & Engineering:
	Walk 35 feet N to (1561, 1682)
	Walk 51 feet E to (1586, 1678)
	Walk 166 feet E to (1666, 1677)
	Walk 43 feet N to (1666, 1656)
	Walk 22 feet E to (1676, 1655)
	Walk 43 feet S to (1677, 1676)
	Walk 141 feet S to (1675, 1743)
	Walk 35 feet E to (1691, 1737)
	Walk 27 feet NE to (1703, 1731)
	Walk 72 feet E to (1737, 1740)
	Walk 27 feet NE to (1749, 1733)
	Walk 59 feet S to (1759, 1760)
	Walk 41 feet S to (1766, 1779)
	Walk 79 feet E to (1801, 1765)
	Walk 45 feet NE to (1821, 1756)
	Walk 136 feet SE to (1852, 1813)
	Walk 84 feet NE to (1889, 1797)
	Walk 151 feet NE to (1952, 1766)
	Walk 103 feet NE to (1996, 1743)
	Walk 134 feet NE to (2055, 1718)
	Walk 21 feet E to (2067, 1714)
	Walk 200 feet NE to (2157, 1675)
	Walk 134 feet E to (2218, 1696)
	Walk 90 feet E to (2261, 1707)
	Walk 18 feet S to (2260, 1716)
Total distance: 1957 feet

Enter an option ('m' to see the menu): 
# All of your test scripts should end with q to terminate the program.